instrumentation/net/http/splunkhttp/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/signalfx/splunk-otel-go/instrumentation/net/http/splunkhttp
 
 go 1.14
 
+replace go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => ../otelhttp
```

Comment:
I have "vendored" `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp` module to easily experiment and visualize the changes in OTel repo